blog/2024-09-17-wasmcloud-postgres-capability-brings-pluggable-databases/index.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "The wasmCloud Postgres capability brings pluggable databases for your apps"
+authors: [ericgregory]
+image: './images/wasmcloud-postgres-capability.webp'
+slug: 2024-09-17-wasmcloud-postgres-capability-brings-pluggable-databases
+date: "2024-09-17"
+description: "Learn how to get started with the Postgres capability in wasmCloud."
+---
+
+![The wasmCloud Postgres capability brings pluggable databases for your apps](./images/wasmcloud-postgres-capability.webp)
+
+With wasmCloud, you can build applications out of WebAssembly components and pluggable, reusable [**capabilities**](/docs/concepts/capabilities), so you can focus on the core logic of your application and abstract away the rest, all using open source standards.
```

Comment:
Might be nice to make `wasmCloud` a link here to the docs/intro -- people *could* land here without knowing what wasmCloud is